tool/tsh/mfa.go

```diff
@@ -112,7 +117,25 @@ func (c *mfaLSCommand) run(cf *CLIConf) error {
 	// Sort by name before printing.
 	sort.Slice(devs, func(i, j int) bool { return devs[i].GetName() < devs[j].GetName() })
 
-	printMFADevices(devs, c.verbose)
+	switch strings.ToLower(c.format) {
+	case teleport.Text:
+		printMFADevices(devs, c.verbose)
+	case teleport.JSON:
+		out, err := utils.FastMarshalIndent(devs, "", "  ")
```

Comment:
Both JSON and YAML will output significantly more details about MFA devices than [`printMFADevices`](https://github.com/gravitational/teleport/blob/7bd7510ff98c71547577eb5fa61c9e37b647a04a/lib/web/ui/mfa.go#L24). If this is intended than it is all good, but I wanted to make sure this is the case.

Reply:
Yes, this is intentional, and it's the case for a lot of the new command outputs. I chose the more verbose output because that's what the commands that already had JSON output do (such as `ls` and `requests ls`).